docs/data/data-grid/performance/performance.md

```diff
@@ -23,8 +79,8 @@ function Component(props) {
 }
 ```
 
-An easy way to prevent re-renders is to extract any object that can be a static object, and to memoize any object that
-depends on another object. This applies to any prop that is an object or a function.
+An easy way to prevent re-renders is to extract any and all objects that can be static, and to memoize any and all objects that depends other objects.
```

Comment:
(Could be just me) but "any and all objects" twice makes it a little confusing:
```suggestion
An easy way to prevent re-renders is to extract any and all objects that can be static, and to memoize those depending on other objects.
```